app/views/full-page-examples/cookie-banner-client-side/index.njk

```diff
@@ -0,0 +1,226 @@
+---
+name: Cookie banner (client side)
+scenario: >-
+  You need to make a choice about whether to accept cookies or not.
+
+notes: >-
+  For this example, when the user makes a choice to accept or reject cookies
+  their preference is handled entirely on the client side with no page
+  navigation, and JavaScript is used to swap out the cookie banner for the
+  'confirmation' banner.
+  
+  The choice to accept or reject cookies will not be remembered.
+
+  The content of the page is not important for this scenario.
+---
+
+{# This example is based of the live "Apply online for a UK passport" start page: https://www.gov.uk/apply-renew-passport #}
+{% extends "_generic.njk" %}
+
+{% from "breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "button/macro.njk" import govukButton %}
+{% from "cookie-banner/macro.njk" import govukCookieBanner %}
+{% from "tabs/macro.njk" import govukTabs %}
+
+{% set pageTitle = "Apply online for a UK passport" %}
+{% block pageTitle %}{{ pageTitle }} - GOV.UK{% endblock %}
+
+{% block bodyStart %}
+    {{ govukCookieBanner({
+    "banners": [
+        {
+            "headingText": "Cookies on this government service",
+            "text": "We use analytics cookies to help understand how users use our service.",
+            "actions": [
+                {
+                    "text": "Accept analytics cookies",
+                    "type": "button",
+                    "name": "cookies",
+                    "value": "accept",
```

Comment:
Do these buttons need a `name` and `value`?

Reply:
Ah, I think this is because we're demonstrating how you might progressively enhance a server side banner? Makes sense 👍

Reply:
Not really, they were just copied and pasted from the server-side example to be honest. Removed in 823bff3.